invenio_communities/members/services/service.py

```diff
@@ -241,15 +242,6 @@ def _add_factory(
         request_id=None,
     ):
         """Add a member to the community."""
-        # TODO: inefficient to do here, should be done in bulk instead.
-        if member["type"] == "group":
```

Comment:
question: where did this code go?

Reply:
This was a hack to map the name to the id. Not any more needed. Now you can have a role with an ID which is different from the name.